sql/testdata/distinct

```diff
@@ -0,0 +1,139 @@
+statement ok
+CREATE TABLE xyz (
+  x INT PRIMARY KEY,
+  y INT,
+  z INT,
+  INDEX foo (z, y)
+)
+
+statement ok
+INSERT INTO xyz VALUES (1, 2, 3), (2, 5, 6), (3, 2, 3), (4, 5, 6), (5, 2, 6), (6, 3, 5), (7, 2, 9);
+
+query II
+SELECT y, z FROM xyz
+----
+2 3
+5 6
+2 3
+5 6
+2 6
+3 5
+2 9
+
+query II
+SELECT DISTINCT y, z FROM xyz
+----
+2 3
+5 6
+2 6
+3 5
+2 9
+
+# TODO(vivek): Use the secondary index. Use distinct in index selection.
+query ITT colnames
+EXPLAIN SELECT DISTINCT y, z FROM xyz
+----
+Level  Type     Description
+0      distinct
+1      scan     xyz@primary
+
+query II
+SELECT DISTINCT y, z FROM xyz ORDER BY z
+----
+2 3
+3 5
+2 6
+5 6
+2 9
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y, z FROM xyz ORDER BY z
+----
+Level  Type     Description
+0      distinct y,z
+1      scan     xyz@foo
+-
+
+query II
+SELECT DISTINCT y, z FROM xyz ORDER BY y
+----
+2 9
+2 6
+2 3
+3 5
+5 6
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y, z FROM xyz ORDER BY y
+----
+Level  Type     Description
+0      distinct y
+1      sort     +y
+2      scan     xyz@foo
+-
+
+query II
+SELECT DISTINCT y, z FROM xyz ORDER BY y, z
+----
+2 3
+2 6
+2 9
+3 5
+5 6
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y, z FROM xyz ORDER BY y, z
+----
+Level  Type     Description
+0      distinct y,z
+1      sort     +y,+z
+2      scan     xyz@foo
+-
+
+query I
+SELECT DISTINCT y + z FROM xyz ORDER by (y + z)
+----
+5
+8
+11
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y + z FROM xyz ORDER BY y + z
+----
+Level  Type     Description
+0      distinct
+1      sort     +y + z
+2      scan     xyz@foo
+-
+
+query II
+SELECT DISTINCT y AS w FROM xyz ORDER by z
+----
+2
+3
+5 
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y AS w FROM xyz ORDER BY z
+----
+Level  Type     Description
+0      distinct
+1      nosort   +z
+2      scan     xyz@foo
+-
+
+query II
+SELECT DISTINCT y AS w FROM xyz ORDER by y
+----
+2
+3
+5
+
+query ITT colnames
+EXPLAIN SELECT DISTINCT y AS w FROM xyz ORDER BY y
+----
+Level  Type     Description
+0      distinct w
+1      sort     +w
+2      scan     xyz@foo
+-
```

Comment:
Oh, this is part of the `scan` description. You should place it on the preceding line. Ditto for line 107 and 123.